### ISSUE TITLE
Forgotten email/ account owner for napari

### ISSUE THREAD

**Issue by sofroniewn** [ContinuumIO/anaconda-issues#12634]
Hello, the [napari](napari.org) team is trying to reclaim ownership of the napari username on https://anaconda.org/napari 

It was registered in April 2019 and has been inactive since then. There’s a good chance it was created by one of the napari steering council members or core developers, but none of us can remember the account details and we don’t seem to be getting the reset emails, cc @royerloic @jni

@jaimergp recommended we reach out here. Please let us know how we should proceed, thanks!!

**Comment by csoja** [comment]
Unfortunately, this looks to be an individual user's account - and was not created by anyone related to the napari team. If you'd like, we can reach out to the current owner to find out if they plan to use it or if we can delete the current account, and we can create an organization with the same name for you to take over.  Please let me know if you'd like us to do that.

**Comment by sofroniewn** [comment]
> if we can delete the current account, and we can create an organization with the same name for you to take over. Please let me know if you'd like us to do that.

Yes, I think that would be great. You can use the "official" `napari-steering-council@googlegroups.com` email address if you need one for the new organization. Thanks so much!!

Unless @jaimergp (who has been advising us in this process) flags otherwise with an alternative recommendation

**Comment by jaimergp** [comment]
Yes, if possible we would like to get `napari` as an organization :) Thanks @csoja! Do you need any login info for anaconda.org? My username there is `jaimergp` as well.

**Comment by sofroniewn** [comment]
My anaconda username is `sofroniewn` if needed
